study.md

```diff
@@ -44,21 +44,24 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+It can store the page value at the time of post back (Sending and Receiving information from Server) of your page. This allows for faster loading as the basics and some complexity of the page are saved.
+
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+Single page applications load faster, are dynamic. The development is simplified and streamlined. There is no need to write code to render pages on the server.
+SPAs are easy to debug with Chrome, as you can monitor network operations, investigate page elements and data associated with it.
+It’s easier to make a mobile application because the developer can reuse the same backend code for web application and native mobile application.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+javascript frameworks such as Ember, Angular, AJAX, JSON, XML and many more.
 ```
```

Comment:
just to clarify, AJAX, JSON, XML are not frameworks, but they can be used to build SPAs. Nice!